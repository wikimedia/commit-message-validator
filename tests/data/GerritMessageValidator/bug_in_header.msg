T1234 Fix

Should fail because a bug id is written in the subject

Change-Id: I395fd614bfa8bcfc46a35f955fb77ec6f03f7a01
