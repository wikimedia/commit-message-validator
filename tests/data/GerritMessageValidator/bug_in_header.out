The following errors were found:
- Line 1: Do not define bug in the subject
